### ISSUE TITLE
robot visual generation

### ISSUE THREAD

**Issue by zhouliang-yu** [siddk/voltron-robotics#6]
Hey @siddk thx so much for sharing this repo, which is really amazing!

I have a question related to the downstream usage of the encoder:
given language description i.e. "peel the carrot" in the demo you show in the paper , and observation at initial state $s_{t = 0}$,
is there any method that I can predict the observation image of the result state $s_{t = k}$?

**Comment by siddk** [comment]
Hey @zhouliang-yu — this is an interesting question. Out of the box, this model doesn’t provide future frame prediction capabilities… however if you wanted to try finetuning/continued pretraining the models, you could imagine feeding the “empty” (fully masked) context for the future frame, and trying to directly regress pixel values. I’m not sure this would work, but could be a decent starting point.
